common/src/main/java/com/amazonaws/services/schemaregistry/common/SchemaV2.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates.
+ * Licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.amazonaws.services.schemaregistry.common;
+
+import lombok.AllArgsConstructor;
+import lombok.Value;
+import software.amazon.awssdk.services.glue.model.Compatibility;
+
+/**
+ * Schema entity represents a schema and it's properties stored in Glue Schema Registry.
+ */
+@AllArgsConstructor
+@Value
+public class SchemaV2 {
```

Comment:
Can we re-use the existing `Schema` class?

Reply:
Removed. Not needed with the new code.